migrations/20201115230456-create-user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      discordId: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
```

Comment:
Just FYI you don't HAVE to have `id` as PK. If you want to rely on `discordId` I think you can just set the discord ID block with `primaryKey: true`.

Reply:
This is intentional, I don't want to use ```discordId``` as PK.